watcher_latency.go

```diff
@@ -14,11 +15,36 @@ func (lw *LatencyWatcher) describe(ch chan<- *prometheus.Desc) {
 }
 
 func (lw *LatencyWatcher) infoKeys() []string {
```

Comment:
I didn't really do a good job of naming this function. The better name would be `passOneKeys`

Reply:
Sounds good. changed `infoKeys()` to `passOneKeys()` and `detailKeys()` to `passTwoKeys()`.